routes/lobby_api.js

```diff
@@ -4,34 +4,53 @@ let isFetching = false;
 
 const PLAYER_COUNT_UPDATE_INTERVAL = parseInt(process.env.PLAYER_COUNT_UPDATE_INTERVAL) * 1000;
 
-exports = module.exports = function(req, res) {
-    const resource = req.query.resource;
-    if (cache[resource]) {
-        if (isFetching || Date.now() - cache[resource].pollTime < PLAYER_COUNT_UPDATE_INTERVAL) {
-            return res.send(cache[resource].count.toString());
-        }
+exports = module.exports = function (req, res) {
+  let resource = req.query.resource;
+  if (cache[resource]) {
+    if (isFetching || Date.now() - cache[resource].pollTime < PLAYER_COUNT_UPDATE_INTERVAL) {
+      return res.send(cache[resource].data);
+    }
+  }
+  isFetching = true;
+  let queryResource = resource;
+  if (resource === "countries") {
+    queryResource = "players";
+  }
+  request(process.env.LOBBY_API_URL + "/" + queryResource, function (error, response, body) {
+    let data = [];
+
+    if (body) {
+      data = JSON.parse(body);
+    } else {
+      console.error(
+        "Error occured during parsing: ",
+        process.env.LOBBY_API_URL + "/" + resource,
+        "body: ", body,
+        "error: ", error,
+        "response", response
+      );
     }
-    isFetching = true;
-    request(process.env.LOBBY_API_URL + "/" + resource, function (error, response, body) {
-        let data = [];
 
-        if (body) {
-            data = JSON.parse(body);
+    cache[queryResource] = {
+      pollTime: Date.now(),
+      data: data.length.toString()
+    };
+    if (queryResource === "players") {
```

Comment:
if we asked for players on the lobby server also calculate/map the countries resource.